CONTRIBUTING.md

```diff
@@ -1,26 +1,82 @@
-Contributing to chef-netuitive
-======================================
+Contributing to Netuitive's Cookbook (Chef)
+==========================================
+Contributions are welcome and can be represented in many different ways as noted below. Help is 
+greatly appreciated and credit will always be given.
 
-### Workflow for contributing
+ 
+Types of Contributions
+------------------------------
+ 
+### Reporting Bugs
+Report bugs on [the issues page](https://github.com/Netuitive/chef-netuitive/issues).
+With your bug report, please include:
+- Your operating system name and version.
+- Any details about your local setup that might be helpful in troubleshooting the issue.
+- Detailed steps to reproduce the bug.
+ 
+### Fixing bugs
+Find bugs at [the issues page](https://github.com/Netuitive/chef-netuitive/issues). Anything tagged with
+"bug" is open to be fixed. 
+With your fix, please include:
+- The issue number
+- A detailed commit message
+ 
+### Implementing Features
+Find features at [the issues page](https://github.com/Netuitive/chef-netuitive/issues). Anything tagged
+with "feature" is open to be implemented.
+With your feature, please include:
+- The issue number
+- A detailed commit message
+ 
+### Writing Documentation
+The Netuitive Cookbook can always use documentation (more documentation is always better!). 
+Please document your features or usage as part of the official docs, in docstrings, 
+in blog posts, articles, or wherever you see fit.
+ 
+### Submitting Feedback
+File an issue at [the issues page](https://github.com/Netuitive/chef-netuitive/issues).
+If you are proposing a feature:
+- Explain how it would work in detail
+- Keep the scope as narrow as possible to make it easier to implement
+ 
+Workflow
+------------------------------
+ 
+1. Create a branch directly in this repo or a fork (if you don't have push access). Please name
+branches within this repository `feature/<description>` or `fix/description`. For example, 
+something like `feature/upgrade_agent_0.2.3-70`.
 
-1. Create a branch directly in this repo or a fork (if you don't have push access). Please name branches within this repository `feature/<description>` or `fix/description`. For example, something like `feature/upgrade_agent_0.2.3-70`.
+2. Create an issue or open a pull request (PR). If you aren't sure your PR will solve the issue
```

Comment:
why manually keep track of numbers in github md format 1. will automatically find the correct number